packages/manager/modules/octavia-load-balancer/src/onboarding/template.html

```diff
@@ -1,3 +1,41 @@
+<div data-ng-if="$ctrl.showNoPrivateNetworkModal">
+    <div class="modal d-block" role="dialog">
+        <div class="modal-dialog modal-dialog-centered modal-dialog-scrollable">
+            <div class="modal-content">
+                <oui-modal
+                    data-type="info"
+                    data-heading="{{:: 'octavia_load_balancer_onboarding_private_network_modal_title' | translate }}"
+                    data-primary-action="$ctrl.goToPrivateNetworkCreation()"
+                    data-primary-label="{{:: 'octavia_load_balancer_onboarding_private_network_modal_title' | translate }}"
+                    data-on-dismiss="$ctrl.dismissPrivateNetworkModal()"
+                >
+                    <span
+                        class="mb-0"
+                        data-translate="octavia_load_balancer_onboarding_private_network_modal_description"
+                    ></span>
+                    <a
+                        class="oui-link_icon"
+                        data-ng-href="{{:: $ctrl.privateNetworkHelpUrl }}"
+                        data-ng-click="$ctrl.trackPrivateNetworkHelp()"
+                        target="_blank"
+                        rel="noopener"
+                    >
+                        <span
+                            data-translate="octavia_load_balancer_onboarding_private_network_modal_more_link"
+                        ></span>
+                    </a>
+
+                    <oui-message data-type="warning" class="mt-4">
+                        <span
+                            data-translate="octavia_load_balancer_onboarding_private_network_modal_message"
+                        ></span>
+                    </oui-message>
+                </oui-modal>
+            </div>
+        </div>
+    </div>
+    <div class="oui-modal-backdrop"></div>
+</div>
```

Comment:
**nitpick:** Since we have dedicated template. We dont need this here

Reply:
Done!